rotary/rules/assembly.smk

```diff
@@ -16,8 +16,7 @@ rule assembly_flye:
         consensus_dir=temp(directory("{sample}/assembly/flye/10-consensus")),
         repeat_dir=temp(directory("{sample}/assembly/flye/20-repeat")),
         contigger_dir=temp(directory("{sample}/assembly/flye/30-contigger")),
-        polishing_dir=temp(directory("{sample}/assembly/flye/40-polishing")),
-
+        polishing_dir=temp(directory("{sample}/assembly/flye/40-polishing")) if config.get("flye_polishing_rounds") > 0 else []
```

Comment:
I made this change to fix a separate issue I ran into during testing -- if polishing is turned off for Flye (e.g., to save speed during testing), then Flye does not produce the output folder `40-polishing`